pkgs/servers/dante/default.nix

```
@@ -11,7 +11,9 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ pam libkrb5 cyrus_sasl miniupnpc ];
 
-  configureFlags = ["--with-libc=libc${stdenv.targetPlatform.extensions.sharedLibrary}"];
+  configureFlags = if !stdenv.isDarwin
```

On linux no config flag is required to build it properly:
```suggestion
  configureFlags = with stdenv; lib.optional isDarwin "--with-libc=libc${stdenv.targetPlatform.extensions.sharedLibrary}";
```